src/atwho.js

```diff
@@ -17,11 +17,14 @@ define([
 
     // stripped down version of http://codepen.io/ImagineProgramming/storydump/javascript-memoization-timeout
     var timed = function timed(f, timeout) {
-        var cache, time = 0;
+        var time = 0;
 
         return function TimedMemoizedFunction() {
             var now = +new Date(),
-                timedOut = (now - time) >= timeout;
+                timedOut = (now - time) >= timeout,
+                form = arguments[0],
+                cache = form.vellum.data.atwho;
+            // atwho isn't actually defined anywhere, but data is an object from core
```

Comment:
`form.vellum.data.atwho` is initialized [here](https://github.com/dimagi/Vellum/blob/32c3d34c99c0d57c098af3bfb2ec58ded4556599/src/base.js#L129). You might want to do something like
```js
cache = form.vellum.data.atwho.cache;
...
// later when refilling the cache
cache = form.vellum.data.atwho.cache = f.apply(f, arguments);
```
as is I'm not sure it's ever updating `form.vellum.data.atwho` with new cache values.